modules/cmd.js

```diff
@@ -0,0 +1,63 @@
+module.exports = async (ctx) => {
```

Comment:
```suggestion
module.exports = async ({msg,bot,members,cfg}) => {
```
use ES6 structuring (or assign shortcuts early) to avoid having to type ctx 60 times throughout this file